packages/iris-grid/src/GotoRow.tsx

```diff
@@ -385,6 +385,7 @@ const GotoRow = forwardRef<GotoRowElement, GotoRowProps>(
 
                 <div>
                   <Button
+                    data-testid="increment-go-to"
```

Comment:
Instead of setting `data-testid`, set the `tooltip` and use that (it automatically sets the `aria-label` property if it's an icon only button). Also need to pass the icon as the `icon` prop instead of a child.:
```suggestion
<Button
  kind="ghost"
  icon={vsArrowUp}
  tooltip="Next match"
  disabled={gotoValue === ''}
  onClick={() => { onGotoValueSubmit(true): }}
  />
```